packages/server/postgres/queries/getTeamPromptResponsesByIds.ts

```diff
@@ -10,7 +10,7 @@ export interface TeamPromptResponse
   content: JSONContent
 }
 
-const mapToTeamPromptResponse = (
+export const mapToTeamPromptResponse = (
```

Comment:
-1 don't do in runtime what you can do in build time. casting a response to a type shouldn't require a function

Reply:
could you please elaborate a bit more? how can we do the casting in the build time? also, this function is heavily updated in https://github.com/ParabolInc/parabol/pull/6407/files#diff-373264581dbe0ee3e09bdbe59799c483616655423e01c4f6b3ecc9338f17a4a0R18-R30 to cast reactjis to proper type so we'd want to keep that in mind

Reply:
after typescript compiles this function, it will just be an identity function where the input is the same as the output. 
instead, you just want to cast:

```ts
type Foo extends Omit<IGetTeamPromptResponsesByIdsQueryResult[0], 'content'> {
  content: JSONContent
}
return responses as Foo[]
```